aospy_synthetic/test/test_objs/models.py

```diff
@@ -1,76 +1,8 @@
 from aospy_synthetic.model import Model
-import cases as runs
-#import idealized
-from aospy_synthetic.db.sqlalchemy.sqlalchemy_db import SQLAlchemyDB
+import runs
```

Comment:
This is a great call -- strip these objects down to the bare minimum needed to test the functionality in question.